lib/ui/HuenicornAppBar.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:flutter/material.dart';
+
+AppBar createAppBar(String title, List <Widget> appBarActions) {
```

Comment:
createAppBar is a weird name for widget. lets rename it to huenicornAppBar

Reply:
in this case I didn't create a widget class. It's a function that returns a widget.
For clarity I can rename it to createHuenicornAppBar